environments/sbox/backend_lb_config.yaml

```diff
@@ -54,6 +54,101 @@ gateways:
       - product: plum
         component: frontend
 
+    # SIDAM 
+      - product: idam
```

Comment:
I'm guessing these were used for testing the module and all should be removed now?

Reply:
they are currently on both gateways but after a successful merge and deploy we will then do another change to split each of the names (SIDAM, Plum Frontend, etc) equally across both of the gateways for the sbox env.

Reply:
@darren221980 lets have a chat about how we will look at the deployment of additional application gateway.